fetch show data

mntedev opened the pull request:
I have completed the following tasks. 
1. added get data function to retrieve shows data from tv maze API.
2. added display shows data feature to display shows data on the home page on card layout 
3. added stsyling for cards

mntedev committed:
add  retrive  all data  function

mntedev committed:
add  tv show rendering section

mntedev committed:
add get tv show function

mntedev committed:
add constants

mntedev committed:
add display and retrive function

mntedev committed:
add card styling

mntedev committed:
add card styling

mntedev committed:
change import to default

mntedev committed:
change export to default

mntedev committed:
remove linters

mntedev committed:
add like structure

mntedev committed:
add like and show card styling

KaskMIL commented on src/module/dom-utils.js:
```diff
@@ -0,0 +1,31 @@
+            <i class="fa fa-heart heart" aria-hidden="true"></i>
+            <div class="likes-num">
+               <span> ${data.length}</span> likes
+            </div>
```
Maybe after the likes-num div you can add a comments button to display the popup section, the rest looks nice

KaskMIL reviewed (changes requested):
Looks so good so far, but maybe we can implement the change I suggest in the comment

mntedev committed:
add comment button

mntedev committed:
add comment button styling

mntedev commented:
I have implemented your change request.

 1. I have added the comment button.
 
![image](https://user-images.githubusercontent.com/66208963/175068724-aaffca4c-0387-419c-aa3d-ed72a2897269.png)

KaskMIL commented:
That looks fantastic!
